presidio-analyzer/presidio_analyzer/nlp_engine/spacy_nlp_engine.py

```diff
@@ -13,14 +13,20 @@ class SpacyNlpEngine(NlpEngine):
         The SpacyNlpEngine uses SpaCy as its NLP module
     """
 
-    def __init__(self):
-        logger.info("Loading NLP model: spaCy en_core_web_lg")
+    engine_name = "spacy"
+    is_available = True if spacy else False
```

Comment:
I use it in the [`__init__`](https://github.com/dhpollack/presidio/blob/dhp/allow_multiple_langs/presidio-analyzer/presidio_analyzer/nlp_engine/__init__.py) of the nlp_engine, so that we know what engines are available.  It also makes it easier to add new engines if someone wanted to do that by having a consistent interface.